resources/rds/v1alpha1/db-security-group.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
This was removed from the CRD: https://github.com/aws-controllers-k8s/rds-controller/pull/80